### DISCUSSION TITLE
Does the word 本 have an English translation in these sentences?

### DISCUSSION THREAD

**Discussion by rodrigo-morales-1** [rodrigo-morales-1/subtitles-npcr#1089] [Q&A]
![image](https://github.com/rdrg109/subtitles-npcr/assets/74389646/5de777be-fee4-4c0a-9093-6549737b0f27)

I don't seem to find an  English translation for 本 in these sentences.

https://github.com/rdrg109/subtitles-npcr/blob/b5d415b7f3aaa0fc4e31696621a845c1f0287a6a/sentences/45-text-1.yaml#L72-L76

https://github.com/rdrg109/subtitles-npcr/blob/b5d415b7f3aaa0fc4e31696621a845c1f0287a6a/sentences/45-text-1.yaml#L277-L281

https://github.com/rdrg109/subtitles-npcr/blob/b5d415b7f3aaa0fc4e31696621a845c1f0287a6a/sentences/45-text-1.yaml#L117-L121

**Comment by fhxshrs** [comment]
“本公司”为"this company"，“本星期”为"this week"，“我本人”为"I"。

**Reply by rodrigo-morales-1** [reply]
Ok! related #1091
